trio/_unix_pipes.py

```diff
@@ -157,3 +160,25 @@ def __init__(self, fd: int):
 
     def fileno(self):
         return self._fd_holder.fd
+
+
+def fd_open_send_stream(fd: int):
```

Comment:
Name bikeshed: I feel like `open_` going at the beginning is maybe an important convention to maintain? Maybe `open_send_stream_to_fd` and `open_receive_stream_from_fd`, or even `open_stream_{to,from}_fd` if we think the send/receive-ness is obvious enough from the to/from-ness.